src/cursor.rs

```diff
@@ -61,7 +64,9 @@ pub trait Cursor<'txn> {
     /// For databases with duplicate data items (`DatabaseFlags::DUP_SORT`), the
     /// duplicate data items of each key will be returned before moving on to
     /// the next key.
-    fn iter_from<K>(&mut self, key: K) -> Iter<'txn> where K: AsRef<[u8]> {
+    fn iter_from<K>(&mut self, key: K) -> Iter<'txn>
```

Comment:
This wrapping is a little too aggressive.  I think

    where_density = "Compressed"

would make it better